CDCgov/NEDSS-Modernization
# Bug/cnft 1423/add sorting to quick condition modal

## Description

Adds the ability to sort by column on the quick conditions modal.  Also had to do a little refactoring to keep QuickConditions and ConditionsLibraryTable more DRY
## Tickets

* [Jira Ticket](https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT2/boards/99?assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4&selectedIssue=CNFT2-1423)

## Steps to verify

1. Navigate to page-builder/manage/pages
2. click Create new page.
3. click "search for condition(s)"
4. Verify sorting by columns works properly.
